### PR TITLE
devopsdays AMS 2016 - Initial commit of speaker info pages

### PR DESCRIPTION
**Pull request by yvovandoorn**

### CODE REVIEW

File: data/speakers/2016/amsterdam/warner-more.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+name: "Warner More"
```
Comment (bridgetkromhout) [inline data/speakers/2016/amsterdam/warner-more.yml]: I think "More" should be "Moore" here and in the filename?

File: data/events/2016-amsterdam.yml [hunk 2]
```diff
@@ -73,29 +73,58 @@ sponsors:
...
   - name: sponsor
   - name: contact
   - name: conduct
+
+speakers:
+  - id: "erica-baker"
+    level: talk
+  - id: "daniel-van-gils"
+    level: talk
+  - id: "warner-more"
```
Comment (bridgetkromhout) [inline data/events/2016-amsterdam.yml]: Spelling?

**Comment by yvovandoorn** [comment]
Yup will fix. Thanks for catching that @bridgetkromhout

**Comment by yvovandoorn** [comment]
Fixed!

**Pull request merged**